src/SFA.DAS.ApplyService.Application/Apply/Moderator/SubmitModeratorOutcomeHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.ApplyService.Domain.Entities;
+
+namespace SFA.DAS.ApplyService.Application.Apply.Moderator
+{
+    public class SubmitModeratorOutcomeHandler : IRequestHandler<SubmitModeratorOutcomeRequest, bool>
+    {
+        private readonly IApplyRepository _repository;
+        private readonly IModeratorRepository _moderatorRepository;
+        private readonly ILogger<SubmitModeratorOutcomeHandler> _logger;
+
+        public SubmitModeratorOutcomeHandler(IApplyRepository repository, IModeratorRepository moderatorRepository, ILogger<SubmitModeratorOutcomeHandler> logger)
+        {
+            _repository = repository;
+            _moderatorRepository = moderatorRepository;
+            _logger = logger;
+        }
+
+        public async Task<bool> Handle(SubmitModeratorOutcomeRequest request, CancellationToken cancellationToken)
+        {
+
+            var applyData = await _repository.GetApplyData(request.ApplicationId);
+
+            var moderatorReviewDetails = applyData?.ModeratorReviewDetails ?? new ModeratorReviewDetails();
```

Comment:
`applyData` shouldn't be null here so you can change it to:
`var moderatorReviewDetails = applyData.ModeratorReviewDetails ?? new ModeratorReviewDetails();`

Will also stop Visual Studio 2019 and SonarCube generating a bug warning 👍